### PR TITLE
added copy button in RollCall QR Token fragment 

### PR DESCRIPTION
**Pull request by Kaz-ookid**
addressing #1916 

Added a copy to clipboard button next to the Token hash

image

### CODE REVIEW

File: docs/messageData.md [hunk 1]
```diff
@@ -3364,6 +3364,7 @@ This message is sent by both organizers to their servers to be broadcast. The me
 
 ```json5
 // ../protocol/examples/messageData/federation_tokens_exchange/federation_tokens_exchange.json
+
 {
```
Comment (matteosz) [inline docs/messageData.md:3368]: There are some unwanted spaces added here I guess

File: docs/messageData.md [hunk 2]
```diff
@@ -3372,11 +3373,13 @@ This message is sent by both organizers to their servers to be broadcast. The me
   "tokens": ["M5ZychEi5rwm22FjwjNuljL1qMJWD2sE7oX9fcHNMDU="],
   "timestamp": 1712854874
 }
+
```
Comment (matteosz) [inline docs/messageData.md:3376]: Same here

File: docs/messageData.md [hunk 3]
```diff
@@ -3372,11 +3373,13 @@ This message is sent by both organizers to their servers to be broadcast. The me
...
 ```
 </details>
 
 ```json5
 // ../protocol/query/method/message/data/dataFederationTokensExchange.json
+
```
Comment (matteosz) [inline docs/messageData.md:3382]: And here

**Review by matteosz — APPROVED** [review]
LGTM! You can even leave those spaces there it's not a big deal, the rest looks great

**Review by quadcopterman — DISMISSED** [review]
Looks good to me

**Comment by Kaz-ookid** [comment]
> LGTM! You can even leave those spaces there it's not a big deal, the rest looks great

Actually those got added by #1950, my checks were failing and I was required to merge it. I don't know what this is about, I've never saw that. But if I remove those spaces, merge will be blocked.

image

**Pull request merged**